src/main/java/com/example/task2/entity/ArticleEntity.java

```diff
@@ -0,0 +1,67 @@
+package com.example.task2.entity;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Entity
+@Table(name = "articles")
+public class ArticleEntity{
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Column(name = "id", updatable = false, unique = true)
+    private UUID id;
+
+    private String name;
+    private String text;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "users_id")
+    private UserEntity userEntity;
+
+    public ArticleEntity() {
```

Comment:
You can use Lombok library and @Data annotation to avoid decalring constructors, getters, setters manually.